code/game/objects/items/weapons/cosmetics.dm

```diff
@@ -558,31 +558,35 @@
 			H.my_appearance.h_style = new_style
 			H.update_hair()
 
-/obj/item/weapon/pocket_mirror/proc/shatter()
+/obj/item/weapon/pocket_mirror/proc/shatter(mob/shatterer)
 	if (shattered)
 		return
 	shattered = 1
 	icon_state = "pocket_mirror_broke"
 	playsound(src, "shatter", 70, 1)
 	desc = "Oh no, seven years of bad luck!"
 
-/obj/item/weapon/pocket_mirror/kick_act()
-	shatter()
+	//Curse the shatterer with bad luck
+	var/datum/blesscurse/brokenmirror/mirrorcurse = new /datum/blesscurse/brokenmirror
+	shatterer.add_blesscurse(mirrorcurse)
+
+/obj/item/weapon/pocket_mirror/kick_act(mob/living/carbon/human/H)
+	shatter(shatterer = H)
```

Comment:
You don't need to specify the var that will receive the ref if there's only one var (or if the vars are given in the same order without skips)